dfttk/pythelec.py

```
@@ -1375,10 +1448,10 @@ def check_vol(self):
             val, idx = min((val, idx) for (idx, val) in enumerate(self.energies))
             if idx!=0 and idx!=len(self.energies)-1: print("\nOK, I found some data that I can try\n")
             else: good = False
- 
+
         if not good:
             print("xxxxxx", self.Vlat, self.volumes)
-            print("\nFATAL ERROR! It appears that the calculations are not all done!\n")
+            print("\nFETAL ERROR! It appears that the calculations are not all done!\n")
```

Mispelling in `FETAL`. This should also raise a proper exception and avoid printing debugging info `print("xxx...`